lib/capistrano/tasks/deploy.rake

```diff
@@ -16,7 +16,9 @@ namespace :deploy do
       invoke 'magento:setup:di:compile'
       invoke 'magento:setup:permissions'
       within current_path do
-        execute :magento, 'maintenance:enable'
+        if test 'ls bin/magento'
```

Comment:
What did you run into where it was _not_ available? I'm curious, because by this point in the script, the mapped command should always be present and ready to run.